runtime/doc/lsp.txt

```diff
@@ -238,6 +238,8 @@ For |lsp-response|, each |lsp-handler| has this signature: >
                                     The parameters used in the original
                                     request which resulted in this handler
                                     call.
+                        {version}   (number) textDocument version (similar to
+                                    |b:changedtick|)
```

Comment:
Since https://github.com/justinmk/neovim/commit/643546b82b4bc0c29ca869f81af868a019723d83 the request handler context (`ctx`) includes a `version` field.

cc @p00f @mfussenegger for sanity check